plugins/woocommerce/tests/e2e-pw/tests/admin-marketing/overview.spec.js

```diff
@@ -11,7 +11,89 @@ test.describe( 'Marketing page', () => {
 
 		// Users should see the "Learn about marketing a store" card.
 		await expect(
-			page.locator( '"Learn about marketing a store"' )
+			page.getByText( 'Learn about marketing a store' )
 		).toBeVisible();
 	} );
+
+	test( 'Marketing Overview page have relevant content', async ( {
+		page,
+	} ) => {
+		// Go to the Marketing page.
+		await page.goto( 'wp-admin/admin.php?page=wc-admin&path=%2Fmarketing' );
+
+		// Heading should be overview
+		await expect(
+			page.getByRole( 'heading', { name: 'Overview' } )
+		).toBeVisible();
+
+		// Call to action is visible
+		await expect(
+			page.getByText( 'Reach new customers and' )
+		).toBeVisible();
+
```

Comment:
Maybe not check this one unless there's an option via the api we can use to ensure it's active?